### DISCUSSION TITLE
Projects on Mobile

### DISCUSSION THREAD

**Discussion by briankelleher** [community/community#41986] [Projects and Issues]
Right now I can't find a way to access or manage GitHub projects on a mobile or tablet device. The option appears to be missing in the mobile app and the mobile website does not allow you to add issues to a project board. Will this become available to smaller devices in the future?

**Comment by jfullstackdev** [comment]
you mean you tried using a mobile browser, like Chrome, then you logged in, and the feature is still not available ? because all the things you can do on GitHub Web can be done through the mobile browser unless it's not supported at the mobile version

**Reply by briankelleher** [reply]
Correct. Project board loads but no way to submit an issue since it's tied to the "enter" key on the desktop, and the standard mobile keyboard does not show that option by default.

**Comment by ghostinhershell** [comment]
Hey there!  👋 

Thanks for posting in the GitHub Community, @briankelleher! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by ghostinhershell** [comment]
Hey again @briankelleher,

> Correct. Project board loads but no way to submit an issue since it's tied to the "enter" key on the desktop, and the standard mobile keyboard does not show that option by default.

I tried this on my end and this is what worked for me: try pressing the SPACE + ENTER button on your mobile keyboard to add new line items to your Project board.

**Reply by MrCatEsq** [reply]
This solution works for me if I do this **in sequence**, but the way looks weird.
